css/global.css

```diff
@@ -45,3 +45,8 @@ body {
 .main-content {
   margin: 0 80px;
 }
+
+
+@media screen and (max-width: 600px) {
+					.main-content {margin: 3%}

```

Comment:
Make sure you don't indent too much and use the right formatting at all times. For example, this code should look like this:

@media screen and (max-width: 600px) {
        .main-content {
                margin: 3%
        }
}